packages/datatrak-web/src/views/AccountSettingsPage/PersonalDetailsSection/PersonalDetailsForm.tsx

```diff
@@ -26,11 +25,19 @@ const StyledTextField = styled(TextField)`
   margin: 0; // Use gap on parent to control spacing
 `;
 
-const PersonalDetailsForm = styled.form`
-  display: grid;
-  gap: 1.56rem 1.25rem;
+const StyledForm = styled(Form)`
   max-width: 44.25rem;
   width: 100%;
+`;
+
+const StyledFieldset = styled.fieldset`
```

Comment:
Hm, would you go as far as splitting the form into multiple `<fieldset>`s? I originally didn’t have one at all; the main reason I’m using it is so that I can apply the `disabled` attribute while the form is being submitted

(`<form>` doesn’t have a `disabled` attribute 😔)

Reply:
Nah I think what you have is good. I agree with the disabled attribute thing as well, it's very annoying not to be able to have an over-arching disabled setting.